css/main.css

```diff
@@ -0,0 +1,248 @@
+/* ------------------Home Page css---------------------- */
+/* PROBLEM: MOBILE VIEW KNOCKS CENTER CONTAINER OUT AND ENDS UP PUSHING NEXT BUTTON OFF SCREEN ON ALL PAGES (need to look at code, and need help with media queries), also ANIMATION MAKES CONTENT POP OUT OF THE BORDER AREA, BUTTON HOVER NOT WORKING*/
+
+/*the body is given a margin of zero to eliminate any margins, and a black background color so entire page appears black*/
```

Comment:
Your overall styling is really good but none of your pages are laid out center since the body styling is changed